core/modules/system/system.api.php

```diff
@@ -797,6 +795,8 @@ function hook_menu_get_item_alter(&$router_item, $path, $original_map) {
  * For a detailed usage example, see page_example.module.
  * For comprehensive documentation on the menu system, see
  * http://drupal.org/node/102338.
+ *
+ * @since 1.24.2 Support for the "position" key removed.
```

Comment:
@BWPanda and @backdrop/core-committers in general, this line should be changed to reflect whichever version this ends up being merged into (1.24.2 or the next minor 1.25.0). Can be done with a commit before merging 😉